runtime/str.go

```diff
@@ -157,6 +157,32 @@ func strAdd(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return NewStr(stringV + stringW).ToObject(), nil
 }
 
+func strCapitalize(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "capitalize", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0])
+	sv := s.Value()
+	numBytes := len(sv)
+	if numBytes == 0 {
+		return s.ToObject(), nil
+	}
+	buf := bytes.Buffer{}
+	if sv[0] >= 'a' && sv[0] <= 'z' {
+		buf.WriteByte(sv[0] - 'a' + 'A')
```

Comment:
You could store a constant with value `'A' - 'a'` and then add and subtract that as necessary.